net/icmpv6/icmpv6.h

```diff
@@ -97,13 +97,19 @@ struct icmpv6_conn_s
    */
 
   struct iob_queue_s readahead;  /* Read-ahead buffering */
+  struct icmp6_filter filter;    /* ICMP6 type filter */
```

Comment:
Question: why `icmp6_filter` is not defined as `icmp6_filter_s` since it is a struct?

Reply:
Because icmp6_filter is public struct defined by FreeBSD/Linux.

Reply:
Understood, thank you @xiaoxiang781216